src/rgw/rgw_sal_motr.cc

```diff
@@ -1906,9 +1906,13 @@ int MotrObject::write_mobj(const DoutPrefixProvider *dpp, bufferlist&& in_buffer
       io_ctxt.total_bufer_sz += left;
       return 0;
     } else {
-      // Append last buffer
-      io_ctxt.accumulated_buffer_list.push_back(std::move(data));
-      io_ctxt.total_bufer_sz += left;
+      // This is last IO. Check if we have previously accumulated buffers.
```

Comment:
when accumulated buffer reaches to size of bs its needs to be written to MOTR immediately before reading next block of data